scripts/patchFiles.ts

```diff
@@ -39,6 +39,22 @@ async function main() {
       }
     })
   }
+
+  // Patch native files for Bugsnag API key
+  const env = require('../env.json')
+  const bugsnagFiles = ['./android/app/src/main/AndroidManifest.xml', './ios/edge/Info.plist']
+
+  for (const file of bugsnagFiles) {
+    await searchReplace(file, 'BUGSNAG_API_KEY', env.BUGSNAG_API_KEY)
+  }
+}
+
+async function searchReplace(file: string, search: string, replace: string): Promise<void> {
+  console.log(`${file} ${search} ${replace}`)
```

Comment:
Should we be logging keys like this? Or have we accepted that the log output for builds are sensitive

Reply:
Yes build logs are sensitive-ish (no private keys though).  Also they never leave the local machine. We already log the deploy-config.json which has some key info